### ISSUE TITLE
Building FatFramework

### ISSUE THREAD

**Issue by nsekhar90** [icerockdev/moko-resources#166]
Hello,

Im trying to build a fatFramework from the multiplatform project. Before using moko resources, my task used to work fine.

```
val fatFrameworkTask = tasks.register(
    "releaseFatFramework",
    org.jetbrains.kotlin.gradle.tasks.FatFrameworkTask::class
) {
    baseName = frameworkName

    from(
        kotlin.iosArm64().binaries.getFramework("RELEASE"),
        kotlin.iosX64().binaries.getFramework("RELEASE")
    )
}
```

With moko, I noticed the sample in the Read.me and modified my task to the following:

```
val fatFrameworkTask = tasks.register("releaseFatFramework", dev.icerock.gradle.generator.FatFrameworkWithResourcesTask::class) {
    baseName = frameworkName

    val targets = mapOf(
        "iosX64" to kotlin.targets.getByName<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget>("iosX64"),
        "iosArm64" to kotlin.targets.getByName<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget>("iosArm64")
    )

    from(
        kotlin.iosArm64().binaries.getFramework("RELEASE"),
        kotlin.iosX64().binaries.getFramework("RELEASE")
    )
}
```

This is what I have in multiplatformResources in gradle

```
multiplatformResources {
    multiplatformResourcesPackage = "com.itps.memxapi.shared"
    disableStaticFrameworkWarning = true
}
```

 When trying to build this I end up with the below error. Am i missing anything ?  
 
 Caused by: org.gradle.api.InvalidUserDataException: Cannot convert URL 'com.norton.kmp.networking:shared.bundle' to a file.
        at org.gradle.api.internal.file.AbstractFileResolver.convertObjectToFile(AbstractFileResolver.java:104)
        at org.gradle.api.internal.file.AbstractBaseDirFileResolver.doResolve(AbstractBaseDirFileResolver.java:39)
        at org.gradle.api.internal.file.AbstractFileResolver.resolve(AbstractFileResolver.java:69)
        at org.gradle.api.internal.file.AbstractFileResolver.resolve(AbstractFileResolver.java:48)
        at org.gradle.api.internal.file.copy.FileCopyAction$FileCopyDetailsInternalAction.processFile(FileCopyAction.java:47)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.maybeVisit(NormalizingCopyActionDecorator.java:100)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.maybeVisit(NormalizingCopyActionDecorator.java:90)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.maybeVisit(NormalizingCopyActionDecorator.java:90)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.maybeVisit(NormalizingCopyActionDecorator.java:90)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.lambda$execute$0(NormalizingCopyActionDecorator.java:67)
        at org.gradle.api.internal.file.copy.DuplicateHandlingCopyActionDecorator.lambda$execute$0(DuplicateHandlingCopyActionDecorator.java:66)
        at org.gradle.api.internal.file.copy.CopyFileVisitorImpl.processFile(CopyFileVisitorImpl.java:64)
        at org.gradle.api.internal.file.copy.CopyFileVisitorImpl.visitFile(CopyFileVisitorImpl.java:48)
        at org.gradle.api.internal.file.collections.DefaultDirectoryWalker$PathVisitor.visitFile(DefaultDirectoryWalker.java:115)
        at org.gradle.api.internal.file.collections.DefaultDirectoryWalker$PathVisitor.visitFile(DefaultDirectoryWalker.java:68)
        at org.gradle.api.internal.file.collections.DefaultDirectoryWalker.walkDir(DefaultDirectoryWalker.java:62)
        at org.gradle.api.internal.file.collections.DirectoryFileTree.walkDir(DirectoryFileTree.java:149)
        at org.gradle.api.internal.file.collections.DirectoryFileTree.visitFrom(DirectoryFileTree.java:127)
        at org.gradle.api.internal.file.collections.DirectoryFileTree.visit(DirectoryFileTree.java:112)
        at org.gradle.api.internal.file.collections.FileTreeAdapter.visit(FileTreeAdapter.java:90)
        at org.gradle.api.internal.file.CompositeFileTree.visit(CompositeFileTree.java:101)
        at org.gradle.api.internal.file.copy.CopySpecActionImpl.execute(CopySpecActionImpl.java:40)
        at org.gradle.api.internal.file.copy.CopySpecActionImpl.execute(CopySpecActionImpl.java:24)
        at org.gradle.api.internal.file.copy.DefaultCopySpec$DefaultCopySpecResolver.walk(DefaultCopySpec.java:756)
        at org.gradle.api.internal.file.copy.DefaultCopySpec$DefaultCopySpecResolver.walk(DefaultCopySpec.java:758)
        at org.gradle.api.internal.file.copy.DefaultCopySpec.walk(DefaultCopySpec.java:498)
        at org.gradle.api.internal.file.copy.DelegatingCopySpecInternal.walk(DelegatingCopySpecInternal.java:282)
        at org.gradle.api.internal.file.copy.CopySpecBackedCopyActionProcessingStream.process(CopySpecBackedCopyActionProcessingStream.java:39)
        at org.gradle.api.internal.file.copy.DuplicateHandlingCopyActionDecorator.lambda$execute$1(DuplicateHandlingCopyActionDecorator.java:43)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.lambda$execute$1(NormalizingCopyActionDecorator.java:60)
        at org.gradle.api.internal.file.copy.FileCopyAction.execute(FileCopyAction.java:38)
        at org.gradle.api.internal.file.copy.NormalizingCopyActionDecorator.execute(NormalizingCopyActionDecorator.java:59)
        at org.gradle.api.internal.file.copy.DuplicateHandlingCopyActionDecorator.execute(DuplicateHandlingCopyActionDecorator.java:43)
        at org.gradle.api.internal.file.copy.CopyActionExecuter.execute(CopyActionExecuter.java:40)
        at org.gradle.api.internal.file.copy.FileCopier.doCopy(FileCopier.java:85)
        at org.gradle.api.internal.file.copy.FileCopier.copy(FileCopier.java:70)

**Comment by nsekhar90** [comment]
I looked at FatFrameworkWithResourcesTask and made some minor modifications which got the task to work and get a fatframework with resources in:

```
open class TestFatFramework : org.jetbrains.kotlin.gradle.tasks.FatFrameworkTask() {

    @TaskAction
    protected fun copyBundle() {
        super.createFatFramework()

        frameworks.first().outputFile.listFiles()
                ?.asSequence()
                ?.filter { it.name.contains(".bundle") }
                ?.forEach { bundleFile ->
                    project.copy {
                        from(bundleFile)
                        into("${fatFrameworkDir.absolutePath}/${bundleFile.name}")
                    }
                }
    }
}
```

**Comment by Alex009** [comment]
fixed in 0.17.3
